extension/src/config/config.js

```diff
@@ -16,17 +16,17 @@ function getCTPEnvCredentials () {
     projectKey: process.env.CTP_PROJECT_KEY,
     clientId: process.env.CTP_CLIENT_ID,
     clientSecret: process.env.CTP_CLIENT_SECRET,
-    apiUrl: 'https://api.sphere.io',
-    authUrl: 'https://auth.sphere.io'
+    apiUrl: process.env.CTP_HOST || 'https://api.sphere.io',
```

Comment:
do we know the region? I'm not which one is default?

Reply:
I guess it depends where you are. Since we're in Europe, I would choose Frankfurt. Any of them is better than `api.sphere.io`, because it is deprecated.

Reply:
Actually it could be Belgium too, so wdyt without using a default value and make it required ?

Reply:
I would put sth there by default. If it's too slow, than the user can choose another API.

Reply:
What is the benefit of having a default value for sth not clear? This is not a batch size so you could not predict it, I would go with the required one so library user should define it like a projectkey and clientId.

Reply:
I'm more for concept of having most of the things by default and the user has to set up minimum stuff.
In this case if you choose Europe and you're in America, it will work anyway without any problems and you don't have to set up. It's not sth that will not work at all. It's just maybe not optimal for some cases. In such situations you change the default.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/232/commits/bc45046b67a4fdb6aae6f4e0512a25470d0e5607

Reply:
related issue: https://github.com/commercetools/commercetools-adyen-integration/issues/173